constructors/library/test/README.md

```diff
@@ -0,0 +1,16 @@
+# Tests
+
+These are the set of tests that verify the behaviour of the JS library generated to support the tiler. They are designed to be used under `node` and `npm` to simulate the behaviour of the library under BigQuery.
```

Comment:
the tiler -> constructors.
This actually has to be updated in skel I think so we don't mess with it.